### PR TITLE
Fix multiline labels in folders

### PR DESCRIPTION
**Pull request by neekless**
Attempt to fix multiline labels in folders being cut off. Use HomeLabelRows since that's what folders use.

Please take a look and see if it works as I'm not pro enough. :/

### CODE REVIEW

**Comment by nyancrimew** [comment]
LGTM, will cherry-pick and test later today

File: src/com/android/launcher3/DeviceProfile.java [hunk 1]
```diff
@@ -484,11 +484,13 @@ private void updateAvailableFolderCellDimensions(DisplayMetrics dm, Resources re
     }
 
     private void updateFolderCellSize(float scale, DisplayMetrics dm, Resources res) {
+        int folderLabelRowCount = prefs.getHomeLabelRows();
```
Comment (nyancrimew) [inline src/com/android/launcher3/DeviceProfile.java]: There was an issue here that `prefs` wasn't a member in this class yet, to get rid of all our instantiations of this all over the class I fixed this and applied your commit internally.

Comment (neekless) [reply src/com/android/launcher3/DeviceProfile.java]: Oh right. I just followed the format in the workspace and all apps sections, I should've known I missed it out.
The fix was to add `LawnchairPreferences prefs = Utilities.getLawnchairPrefs(mContext);` correct?

Comment (nyancrimew) [reply src/com/android/launcher3/DeviceProfile.java]: Yup that would have fixed it!

**Pull request closed without merging**